routers/init.go

```diff
@@ -194,5 +201,11 @@ func NormalRoutes(ctx context.Context) *web.Route {
 		// This implements the OCI API (Note this is not preceded by /api but is instead /v2)
 		r.Mount("/v2", packages_router.ContainerRoutes(ctx))
 	}
+
+	if setting.Actions.Enabled {
+		prefix := "/api/actions"
```

Comment:
No, there's no hardcode.

I mean to, because they're quite different.

| `/api/v1` | `/api/actions` |
| --- | --- |
| For general purposes | For actions runners only |
| Can be called by any HTTP client | Can be called by gRPC clients only |
| RESTful, uses kinds of HTTP methods, uses `v1` at prefix of routers | Not RESTful, uses POST only, the routers look like `/api/actions/runner.v1.RunnerService/FetchTask` |
| May be `/api/v2` after an upgrade | May be `/api/actions/runner.v2.RunnerService/FetchTask` |